dev-js-build.js

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+
+const esbuild = require("esbuild");
+const expand = require('glob-expand')
+let entrypoints = expand("app/javascript/**/*.js")
```

Comment:
fyi there is no need to use `let` here, `let` is only necessary if you are going to reassign the variable